src/server/svr_recov_db.c

```diff
@@ -504,8 +504,10 @@ sched_save_db(pbs_sched *ps, int mode)
 		attr_info.parent_obj_type = PARENT_TYPE_SCHED; /* svr attr */
 		attr_info.parent_id = ps->sc_name;
 
+		ps->sch_attr[SCHED_ATR_sched_state].at_flags &= ~(ATR_VFLAG_MODIFY | ATR_VFLAG_MODCACHE);
 		if (save_attr_db(conn, &attr_info, sched_attr_def, ps->sch_attr, (int)SCHED_ATR_LAST, flag) !=0)
 			goto db_err;
+		ps->sch_attr[SCHED_ATR_sched_state].at_flags &= (ATR_VFLAG_MODIFY | ATR_VFLAG_MODCACHE);
```

Comment:
Did you mean |= ?  This will get rid of all bits other than modify and modcache

Reply:
Implemented.